examples/visual-tests/amp-story/amp-story-page-attachment-v2.html

```diff
@@ -129,60 +129,60 @@ <h1>Some really cool video</h1>
       </amp-story-page>
 
       <amp-story-page id="outlink-default">
-        <amp-story-page-attachment layout="nodisplay" href="https://google.com">
-        </amp-story-page-attachment>
+        <amp-story-page-outlink layout="nodisplay" href="https://google.com">
```

Comment:
This one is invalid but works for backwards compatibility for `amp-story-page-attachment[href]`, right?

Reply:
Thanks for catching this. 
I added a visual test as the last page of this template as an example of backwards compatibility.

A note that `amp-story-page-outlink` will not work if it has an href attribute. It must have an anchor child to render.